content/docs/api_reference/config/v1alpha1.md

```diff
@@ -1070,5 +1064,5 @@ for all inbound and ingress traffic in the mesh.</p>
 <hr/>
 <p><em>
 Generated with <code>gen-crd-api-reference-docs</code>
-on git commit <code>386d92a1</code>.
+on git commit <code>11d0c2df</code>.
```

Comment:
Usually I'd say let's wait to generate the docs until the v1.1.0 tag is created on the OSM repo so then this hash would match the tag's commit. These docs aren't going to change though before then so practically it doesn't matter.